aem/build.gradle.kts

```diff
@@ -52,14 +67,21 @@ aem {
             dependsOn(":develop")
         }
 
+        environmentDev {
+            reloader {
```

Comment:
Still, I suggest having this config folder directly in container configuration so we would stick to declarative model instead of task configuration

Reply:
Final idea"

```kotlin
"httpd" {
  dev {
    watchDir("conf")
  }
}
```